src/sage/rings/power_series_ring_element.pyx

```diff
@@ -1384,6 +1384,88 @@ cdef class PowerSeries(AlgebraElement):
             serie = (u - 1 - A * t) / (B * t ** 2)
         return tuple(resu)
 
+    def super_delta_fraction(self, delta):
+        r"""
+        Return the super delta continued fraction of ``self``.
+
+        This is a continued fraction of the following shape:
+
+        .. MATH::
+
+            \cfrac{v_0 x^{k_0}} {U_1(x) -
+            \cfrac{v_1 x^{k_0 + k_1 + \delta}} {U_2(x) -
+            \cfrac{v_2 x^{k_0 + k_1 + k_2 + \delta}} {U_3(x) - \cdots} } }
+
+        where each `U_j(x) = 1 + u_j(x) x`.
+
+        INPUT:
+
+        - ``delta`` -- positive integer, most usually 2
+
```

Comment:
"most usually" sounds clumsy to me.

Any difference between "usually" and "most usually"?

How about
```
positive integer, usually 2
```
?